Cargo.toml

```diff
@@ -16,4 +16,8 @@ travis-ci = { repository = "jethrogb/rust-cexpr" }
 nom = { version = "7", default-features = false, features = ["std"] }
 
 [dev-dependencies]
-clang-sys = ">= 0.13.0, < 0.29.0"
+clang = { version = "2", features = ["runtime"] }
+
+[patch.crates-io]
+clang-sys = { git = "https://github.com/reitermarkus/clang-sys", branch = "load-api" }
```

Comment:
Can you upstream or remove this? It seems unfortunate not depending on the crates.io version.

Reply:
An upstream PR is already opened. This is only needed for the last commit in this PR, cleaning up the test setup.

Reply:
Would it be possible to split the test setup cleanup into its own PR? Seems like there's some push back against making the change to `clang-sys` since it's a breaking change...